src/components/standup/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC } from 'react';
+
+import styles from '@/components/standup/standupContainer.module.scss';
+import { Standup } from '@/interfaces/standup.type';
+
+const StandUpContainer: FC<Standup> = ({
+    handleFormSubmission,
+    handleChange,
+    yesterdayDate,
+    buttonDisable,
+    completed,
+    planned,
+    blockers,
+}: Standup) => {
+    return (
+        <>
+            <section className="container">
+                <section className={styles.standupContainer}>
+                    <section className={styles.standupBanner}>
+                        <p>
+                            You have
+                            <span data-testid="missed-updates">2 missed</span>
+                            Standup updates this week
+                        </p>
+                        <p>Let&apos;s try to avoid having zero days </p>
+                    </section>
+                    <section className={styles.standupUpdateContainer}>
+                        <h1>Standup Update</h1>
+                        <form
+                            className={styles.standupForm}
+                            onSubmit={(e: React.ChangeEvent<HTMLFormElement>) =>
+                                handleFormSubmission(e)
+                            }
+                        >
+                            <section className={styles.yesterdayUpdate}>
+                                <label
+                                    className={styles.updateHeading}
+                                    htmlFor="completed"
+                                >
+                                    On {yesterdayDate}
+                                </label>
+                                <input
```

Comment:
why not make it a Custom component and send all these props through it? You can resolve this comment in case you find it not so intuitive